src-docs/src/views/code/code_example.js

```diff
@@ -39,6 +41,23 @@ export const CodeExample = {
   title: 'Code',
   intro: (
     <>
+      <EuiCallOut title="EUI provides support for static code">
```

Comment:
```suggestion
      <EuiCallOut>
```

I've been leaning towards brevity with callouts so if the description repeats the title, I just like to remove the title.